frontend/src/router/guards.js

```diff
@@ -6,13 +6,17 @@
 
 import includes from 'lodash/includes'
 import isEmpty from 'lodash/isEmpty'
-import { getPrivileges, getConfiguration } from '@/utils/api'
+import {
+  getPrivileges,
+  getConfiguration,
+  getPublicConfiguration
+} from '@/utils/api'
 
 export default function createGuards (store, userManager, localStorage) {
   return {
     beforeEach: [
       setLoading(store, true),
-      ensureConfigurationLoaded(store),
+      ensureConfigurationLoaded(store, userManager),
```

Comment:
We can move this down if we have a static asset for public config

Reply:
✅